frontend/server/src/DAO/Assignments.php

```diff
@@ -115,7 +119,7 @@ public static function getAssignmentForProblemset($problemset_id) {
                     ps.problemset_id = a.problemset_id
                 WHERE
                     a.assignment_id = ? LIMIT 1;';
-
+        /** @var null|array{scoreboard_url: string, scoreboard_url_admin: string} */
         $rs = \OmegaUp\MySQLConnection::getInstance()->GetRow(
```

Comment:
```suggestion
        return \OmegaUp\MySQLConnection::getInstance()->GetRow(
```

esta expresión ya es del tipo correcto, entonces no necesitas L127-L130!

Reply:
Listo